### ISSUE TITLE
forceignore does not work with SFDX: Retrieve Source in Manifest from Org

### ISSUE THREAD

**Issue by jasonmatos** [forcedotcom/salesforcedx-vscode#3236]
### Summary

The .fourceignore file does not seems to be considered when using the VSCode command SFDX: Retrieve Source in Manifest from Org.

### Steps To Reproduce:

1.  Open Salesforce project in VSCode 
2. Open manifest/package.xml file
3. Right click in the package.xml file
4. Select the command SFDX: Retrieve Source in Manifest from Org 

### Expected result

Files listed in the .forceignore should not be retrieve
In this example Integration.profile-meta.xml should not be listed 

### Actual result

All the file are been retrieved from the Org.
In this example all the profiles has been retrieved.

### Additional information

If the following command in the terminal is run. The forceignore is considered.
`sfdx force:source:retrieve -x manifest/package.xml`

Basically, this is my workaround since the SFDX command does not work.

Manifest 
```
<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
<Package xmlns="http://soap.sforce.com/2006/04/metadata">
    <types>
		<members>*</members>
		<name>Profile</name>
	</types>
    <version>51.0</version>
</Package>
```

.forceignore file
```
#.forceignore v2

# List files or directories below to ignore them when running force:source:push, force:source:pull, and force:source:status
# More information: https://developer.salesforce.com/docs/atlas.en-us.sfdx_dev.meta/sfdx_dev/sfdx_dev_exclude_source.htm
#
package.xml

# LWC configuration files
**/jsconfig.json
**/.eslintrc.json

# LWC Jest
**/__tests__/**

force-app/main/default/profiles/Integration.profile-meta.xml
```

**VS Code Version**:
VSCode version 1.56

**SFDX CLI Version**:
sfdx --version
sfdx-cli/7.99.0 win32-x64 node-v15.14.0sf

$ sfdx plugins --core
@oclif/plugin-autocomplete 0.3.0 (core)
@oclif/plugin-commands 1.3.0 (core)
@oclif/plugin-help 3.2.2 (core)
@oclif/plugin-not-found 1.2.4 (core)
@oclif/plugin-plugins 1.10.0 (core)
@oclif/plugin-update 1.4.0-3 (core)
@oclif/plugin-warn-if-update-available 1.7.0 (core)
@oclif/plugin-which 1.0.3 (core)
@salesforce/sfdx-trust 3.6.0 (core)
alias 1.1.9 (core)
auth 1.5.3 (core)
config 1.2.8 (core)
generator 1.1.5 (core)
salesforcedx 51.5.0 (51.5.0)
├─ limits 1.0.4
├─ schema 1.0.4
├─ org 1.5.0
├─ apex 0.1.4
├─ custom-metadata 1.0.11
├─ templates 51.3.0
├─ user 1.2.0
├─ @salesforce/sfdx-plugin-lwc-test 0.1.7
└─ salesforce-alm 51.6.3
sfdx-cli 7.99.0 (core)
telemetry 1.1.2 (core)

**OS and version**:
Windows 10

**Comment by violetyao** [comment]
Hi @jasonmatos, thanks for reporting this! We were not able to reproduce the profile example with the extension version you used. Did `.forceIgnore` work for retrieving other metadata types with manifest? There were a few changes for `.forceIgnore` behaviors since `v51.5.0`. Does bumping up the extension version help?

**Comment by floralan** [comment]
@jasonmatos We have fixed this issue in here [#413](https://github.com/forcedotcom/source-deploy-retrieve/pull/413). Please check out our latest version and you can always open a new issue if you have more questions.
